### PR TITLE
Storage import

### PR DESCRIPTION
**Pull request by crsndr**
# Summary

This new section add a description for the storage import on DataHub and address one issue experienced with in AI4PublicPolicy where changes made to the DataHub space from Notebooks was not reflected on DataHub, leading to inconsistencies

**Related issue :**

https://ggus.eu/index.php?mode=ticket_info&ticket_id=160044

### CODE REVIEW

File: content/en/providers/datahub/oneprovider/_index.md [hunk 1]
```diff
@@ -127,3 +127,52 @@ You can monitor the import activity from the administration panel at port 9443.
...
+deciding to enable the automatic scan of the storage or to perform a single
+scan:
+
+1. An initially empty space, modified only via DataHub -> no need to enable the
+   autoscan or to perform a manual scan.
+1. A space with some initial data that has been imported once and then the space
+   is only modified via DataHub -> In this case the auto scan is not needed
+   however it needs to somehow guard the storage from external modifications
+   otherwise it will desync with the space. E.g.: all changes should be done
+   trough API, DataHub client or the weg interface.
```
Comment (andrea-manzi) [inline content/en/providers/datahub/oneprovider/_index.md:157]:
```suggestion
   trough API, DataHub client or the web interface.
```

File: content/en/providers/datahub/oneprovider/_index.md [hunk 2]
```diff
@@ -127,3 +127,52 @@ You can monitor the import activity from the administration panel at port 9443.
...
+1. A space that exposes a dataset from a storage that later is still dynamically
+   changing outside of DataHub (when it sis assumed that some modification can
```
Comment (andrea-manzi) [inline content/en/providers/datahub/oneprovider/_index.md:159]:
```suggestion
   changing outside of DataHub (when it is assumed that some modification can
```

File: content/en/providers/datahub/oneprovider/_index.md [hunk 3]
```diff
@@ -127,3 +127,52 @@ You can monitor the import activity from the administration panel at port 9443.
...
+and hundreds of changes per second. Otherwise, having the continuous scan run in
+the background every now and then is not a problem. You can also adjust the scan
+interval if needed.
+
+In the following screenshot is shown how to access the configuration for the
+storage import:
+
+![image](storage-import-01.png)
+
+Thois configuration page is located under the "CLUSTER" tab in the main page.
```
Comment (andrea-manzi) [inline content/en/providers/datahub/oneprovider/_index.md:174]:
```suggestion
This configuration page is located under the "CLUSTER" tab in the main page.
```

File: content/en/providers/datahub/oneprovider/_index.md [hunk 4]
```diff
@@ -127,3 +127,52 @@ You can monitor the import activity from the administration panel at port 9443.
...
+   otherwise it will desync with the space. E.g.: all changes should be done
+   trough API, DataHub client or the weg interface.
+1. A space that exposes a dataset from a storage that later is still dynamically
+   changing outside of DataHub (when it sis assumed that some modification can
+   happen at some point) -> in this case continuous scan is needed.
+
+Every storage import scan causes additional load on the Oneprovider. If the
+space is large and changes very often outside of Onedata, it might have a
+visible impact on the overall performance - but we are talking millions of files
+and hundreds of changes per second. Otherwise, having the continuous scan run in
```
[the remark this answers is not in the document]
Comment (crsndr) [reply content/en/providers/datahub/oneprovider/_index.md:165]: I've rephrased this

File: content/en/providers/datahub/oneprovider/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (crsndr) [reply content/en/providers/datahub/oneprovider/_index.md]: this should be correct now

**Review by andrea-manzi — APPROVED** [review]
thanks @crsndr , LGTM!

**Pull request merged**